### ISSUE TITLE
[Accessibility]A11y_AzureDataStudio_CSVPreviewData_ScreenReader: Screen reader is not announcing row and column reference for data cell in the grid.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [microsoft/azuredatastudio#6074]
[Check out Keros!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.

GitHubTags: #A11yMAS; #A11yTCS; #MAS1.3.1; #A11y_AzureDataStudio; #Win32; #DesktopApp

**Environment Details:**
Application name: Azure Data Studio
Version: 1.9.0-insider (user setup)
Electron: 3.1.8
Node.js: 10.2.0
V8: 6.6.346.32
OS: Windows_NT x64 10.0.17763
JAWS version : 2019.1810.64 ILM

**Additional Details:**
Tool Name: JAWS
MAS Reference:MAS1.3.1

**Steps to Reproduce:**
1. Install_ the app Azure Data Studio.
2. Connect to database->navigate to the Data services folder->HDFS Folder-> navigate to CSV files folder->Right click on the files->Select option Create external table from CSV->Click on the "Next" button.
3. Start JAWS  .
4. Use basic arrow navigation to move on the table that is downward arrow to move on next item and up arrow to move on previous item.
5. Observe that Screen reader is not announcing row and column reference for data cell in the grid.

**Actual:**
Using basic arrow navigation, screen reader only announces table item name.
For example, if data item is "Test_Case_Title" screen reader only announce "Test_Case_Title". 
If using Ctrl+Shift+arrows then screen reader announce "Test_Case_Title  column 1", but using Ctrl+Shift+arrows only table header can be accessed and not the other data cell present in the table.
To move on to the  other data cell ,need to use only basic arrow navigation.

**Similar issue also observed with NVDA
Similar issue is also observed in following pages :
1.CSVSummary:** Connect to database->navigate to the Data services folder->HDFS Folder-> navigate to CSV files folder->Right click on the files->Select option Create external table from CSV->Click on the "Next" button->click on next button.

**Expected:**
Screen reader should announces "Test_Case_Title  column 1 row 1" for basic arrow navigation.

**Recommendation:**
Please refer the following snippet link for fixes:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
Screen reader only users find it difficult to understand to navigate through the table because he won't understand on which column or row he is navigating .

**MAS Reference:
MAS1.3.1-**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**Attachment for Reference:**
[6074_[Accessibility]A11y_AzureDataStudio_CSVPreviewData_ScreenReader.pptx](https://github.com/microsoft/azuredatastudio/files/3323712/6074_.Accessibility.A11y_AzureDataStudio_CSVPreviewData_ScreenReader.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by Charles-Gagnon** [comment]
The backing grid has been changed and seems to be announcing the header row/column correctly now using NVDA.
